src/GameRulesInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Spaceship;
+
+interface GameRulesInterface
```

Comment:
This interface is correct, but will you need it? Will there be many types of GameRules?

Reply:
I wasn't sure if I needed it or not, it was more to see if I could have used an interface or not. I could use an abstract class. Or do you suggest just to use a normal class?